app/src/main/java/com/github/multimatum_team/multimatum/procrastination_detector/ProcrastinationDetectorService.kt

```diff
@@ -0,0 +1,82 @@
+package com.github.multimatum_team.multimatum.procrastination_detector
+
+import android.app.Service
+import android.content.Intent
+import android.hardware.Sensor
+import android.hardware.SensorEvent
+import android.hardware.SensorEventListener
+import android.hardware.SensorManager
+import android.os.IBinder
+import android.widget.Toast
+import com.github.multimatum_team.multimatum.R
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+import kotlin.math.abs
+
+@AndroidEntryPoint
+class ProcrastinationDetectorService : Service(), SensorEventListener {
```

Comment:
It should be nice to add a description of the class and multiples comments before the function to explain what they do